client/src/components/landing/RegisterButton.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <button
+      v-if="buttonNumber === 2"
+      class="btn btn-primary"
+      @click="register()">
+      {{ $t('LandingPage.Nav.EndingPageButton') }}
+    </button>
+    <button
+      v-else
+      class="btn btn-primary"
+      @click="register()">
+      {{ $t('LandingPage.Nav.GetStarted') }}
+    </button>
+  </div>
```

Comment:
TBH, the buttonNumber is not clear for me. I would use a `v-slot` in the button, that can be changed, or add just property `text`. `buttonNumber` is a bit odd variable name, what if someone gives "3333"? :D And nothing apart from the translation changes.